domains/tat2008.json

```diff
@@ -0,0 +1,12 @@
+
+    {
+      "owner": {
+        "username": "tat2008",
+        "email": "tuan.trananh2008@gmail.com"
+      },
+
+      "record": {
+        "URL": "tat2008.github.io"
```

Comment:
Did you mean to put ``CNAME`` here? If it's a URL record, it will just redirect to the page. (also make sure to put ``https://`` in the record link, if it's a URL record.

Reply:
oh ok, i will change